<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportLinkDialog</class>
 <widget class="QWidget" name="ImportLinkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>390</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;File with links to import:</string>
       </property>
       <property name="buddy">
        <cstring>urlRequester</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KUrlRequester" name="urlRequester"/>
     </item>
     <item>
      <widget class="KPushButton" name="importLinks">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Import Links</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="KSeparator" name="kseparator_2"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Show:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="showAll">
       <property name="text">
        <string extracomment="filter: show all file types">&amp;All</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>filterButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="showVideos">
       <property name="text">
        <string>&amp;Videos</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>filterButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="showImages">
       <property name="text">
        <string>Ima&amp;ges</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>filterButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="showAudio">
       <property name="text">
        <string>A&amp;udio</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>filterButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="showArchives">
       <property name="text">
        <string>A&amp;rchives</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string>filterButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="textFilter">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>You can use wildcards for filtering.</string>
       </property>
       <property name="clickMessage">
        <string>Filter files here....</string>
       </property>
       <property name="showClearButton" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="filterMode">
       <item>
        <property name="text">
         <string>Contains</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Does Not Contain</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="KPushButton" name="deselectAll">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>D&amp;eselect All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="selectAll">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Select All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="invertSelection">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Inver&amp;t Selection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="checkSelected">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string extracomment="Check as in mark, not as in verify">Chec&amp;k Selected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showWebContent">
       <property name="text">
        <string>Show &amp;web content</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="KSeparator" name="kseparator"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KSeparator</class>
   <extends>QFrame</extends>
   <header>kseparator.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>urlRequester</tabstop>
  <tabstop>importLinks</tabstop>
  <tabstop>textFilter</tabstop>
  <tabstop>filterMode</tabstop>
  <tabstop>showAll</tabstop>
  <tabstop>showVideos</tabstop>
  <tabstop>showImages</tabstop>
  <tabstop>showAudio</tabstop>
  <tabstop>showArchives</tabstop>
  <tabstop>deselectAll</tabstop>
  <tabstop>selectAll</tabstop>
  <tabstop>invertSelection</tabstop>
  <tabstop>checkSelected</tabstop>
  <tabstop>showWebContent</tabstop>
  <tabstop>treeView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="filterButtonGroup"/>
 </buttongroups>
</ui>
